test/unit/test_expid.py

```diff
@@ -57,6 +64,20 @@ def test_create_new_operational_experiment_with_previous_one(self, db_common_moc
         experiment_id = new_experiment(self.description, self.version, False, True)
         self.assertEqual("o113", experiment_id)
 
+    @patch('autosubmit.experiment.experiment_common.db_common')
+    def test_create_new_evaluation_experiment_with_previous_one(self, db_common_mock):
+        current_experiment_id = "e113"
+        self._build_db_mock(current_experiment_id, db_common_mock)
+        experiment_id = new_experiment(self.description, self.version, False, False, True)
+        self.assertEqual("e113", experiment_id)
+
+    @patch('autosubmit.experiment.experiment_common.db_common')
+    def test_copy_experiment_new(self, db_common_mock):
+        current_experiment_id = "empty"
+        self._build_db_mock(current_experiment_id, db_common_mock)
+        experiment_id = copy_experiment(current_experiment_id, self.description, self.version, False, False, True)
+        self.assertEqual("", experiment_id)
+    
     @staticmethod
     def _build_db_mock(current_experiment_id, mock_db_common):
         mock_db_common.last_name_used = Mock(return_value=current_experiment_id)
```

Comment:
There's another partial covered statement (screenshot), which you can cover by testing that the empty last expid returns an empty one too.

![image](https://github.com/user-attachments/assets/70bb0d89-ded2-4863-9ef5-70df78fbd551)


```python
    @patch('autosubmit.experiment.experiment_common.db_common')
    def test_create_new_evaluation_experiment_with_empty_current(self, db_common_mock):
        current_experiment_id = ""
        self._build_db_mock(current_experiment_id, db_common_mock)
        experiment_id = new_experiment(self.description, self.version, False, False, True)
        self.assertEqual("", experiment_id)
```